pkg/controller/pod_volume_backup_controller.go

```diff
@@ -211,7 +211,7 @@ func (r *PodVolumeBackupReconciler) singlePathMatch(path string) (string, error)
 	}
 
 	if len(matches) != 1 {
-		return "", errors.Errorf("expected one matching path, got %d", len(matches))
+		return "", errors.Errorf("expected one matching path: %s, got %d", path, len(matches))
```

Comment:
Have you checked the value of `path` I recall it has the wildcard, what does it look like?

Reply:
``` go
	volDir, err := kube.GetVolumeDirectory(ctx, log, pod, pvb.Spec.Volume, r.Client)
	if err != nil {
		return nil, errors.Wrap(err, "getting volume directory name")
	}

	pathGlob := fmt.Sprintf("/host_pods/%s/volumes/*/%s", string(pvb.Spec.Pod.UID), volDir)
	log.WithField("pathGlob", pathGlob).Debug("Looking for path matching glob")
```
This is the code in PVB controller preparing path before calling the function singlePathMatch.
It has a wildcard, but the volumeDir is also included.